xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEResamplePi.cpp

```diff
@@ -280,7 +280,7 @@ bool CActiveAEResamplePi::Init(uint64_t dst_chan_layout, int dst_channels, int d
 
   LOGTIME(3);
 
-  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock") && CSettings::Get().GetInt("videoplayer.synctype") == SYNC_RESAMPLE)
+  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock"))
```

Comment:
I don't know why this is needed at all. @popcornmix AE does not instantiate a resampler for any case a resampler is not needed. why did you add this check?